src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/enums/Classifications.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmcts.reform.ccd.document.am.model.enums;
+
+public enum Classifications {
```

Comment:
How are Classifications and SecurityClassification different? Names should be self descriptive to distinguish the 2.